app/src/main/java/com/example/stretchy/database/MockedDataBaseImpl.kt

```diff
@@ -2,8 +2,9 @@ package com.example.stretchy.database
 
 import com.example.stretchy.database.data.ExerciseItemDb
 import com.example.stretchy.features.traininglist.ui.data.Training
+import javax.inject.Inject
 
-class MockedDataBaseImpl : DataBase {
+class MockedDataBaseImpl @Inject constructor() : DataBase {
     override suspend fun getTrainings(): List<Training> {
```

Comment:
let's do not use @Injest on constructor, if we have classes that can be constructed in the `Module` - do so.
we can use @Inject in classes that were constructed by Android system `Activities` `Fragments` etc.